src/Microsoft.DotNet.Tools.Publish/Program.cs

```diff
@@ -170,8 +175,34 @@ private static int PublishHost(ProjectContext context, string outputPath)
 
             // Copy the host
             File.Copy(hostPath, outputExe, overwrite: true);
+
+            // Publish the runtime
+            var runtimePath = Path.GetFullPath(Path.Combine(AppContext.BaseDirectory, "..", "runtime", "coreclr"));
+            if(!Directory.Exists(runtimePath))
```

Comment:
Correct. I'm not sure the right approach there since I'm really not sure having a dependency on the Runtime is the right way to do this anyway. Apps shouldn't be dependent on specific runtime versions as much as possible, that should be a deployment-time decision.

I considered having publish take a `--runtime-path` switch to allow the caller to set the source path for runtime files.